README.md

```diff
@@ -0,0 +1,59 @@
+<h1>Running byedpi and socks5 proxy together</h1>
```

Comment:
That header is way tooooooooooooooooooooooooooo long. Would be much better just to replace it with just project name, the description below explains the details.